test/main.test.js

```diff
@@ -189,7 +189,7 @@ test.cb('<!-- AUTO-GENERATED-CONTENT:START (CODE)-->', t => {
     // check local code with range lines
     t.regex(newContent, /```js\n  const baz = 'foobar'\n  console\.log\(`Hello \${baz}`\)\n```/, 'local code snippet with range lines inserted')
     // check remotely fetched code
-    t.regex(newContent, /require\('dox'\)/, 'remote code snippet inserted')
+    t.regex(newContent, /require\('doxxx'\)/, 'remote code snippet inserted')
```

Comment:
Unrelated, but required for tests to pass:
https://github.com/DavidWells/markdown-magic/blob/0cc7e597f5f67ae9577ec386a99038cfa2531312/examples/generate-readme.js#L18